### PR TITLE
Align PHP minimum version checks

### PR DESCRIPTION
**Pull request by radeno**
### Description of the Change

After merging https://github.com/10up/windows-azure-storage/commit/3b7f2fa776cb5a3d2111ce94258245971a4c1517 there was introduced helper method. Align PHP version checking

Closes #

### Changelog Entry

> Added - New feature
> Changed - Existing functionality
> Deprecated - Soon-to-be removed feature
> Removed - Feature
> Fixed - Bug fix
> Security - Vulnerability

### Credits

Props @username, @username2, ...

### Checklist:

- [x] I agree to follow this project's [**Code of Conduct**](https://github.com/10up/.github/blob/trunk/CODE_OF_CONDUCT.md).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [x] All new and existing tests pass.

### CODE REVIEW

**Comment by jeffpaul** [comment]
@10up/open-source-practice could use a quick code review on this for the 4.4.0 release, thanks!

File: windows-azure-storage.php [hunk 1]
```diff
@@ -214,11 +214,9 @@ function windows_azure_storage_load_textdomain() {
  */
 function windows_azure_plugin_check_prerequisite() {
 	global $wp_version;
-	$php_version = phpversion();
-	$php_compat  = version_compare( $php_version, '8.0.0', '>=' );
-	if ( ! $php_compat ) {
+	if ( ! was_minimum_php_requirement() ) {
```
Comment (ravinderk) [inline windows-azure-storage.php]: @radeno `was_minimum_php_requirement ` on return php version. It does not compare the required PHP version with the environment PHP version. Can you add PHP version validation?

Comment (ravinderk) [reply windows-azure-storage.php]: I will suggest to use https://github.com/10up/wp-compat-validation-tool

**Review by ravinderk — CHANGES_REQUESTED** [review]
@radeno, I left a few suggestions.  LEt me know if you have any questions.

**Comment by faisal-alvi** [comment]
@jeffpaul @ravinderk  Should we use https://github.com/10up/wp-compat-validation-tool instead of adding manual checks across all plugins? There is `set_php_min_required_version( 8.0 )` function that seems useful here.

**Comment by jeffpaul** [comment]
@faisal-alvi yes, probably worth updating to that newly available tool

File: windows-azure-storage.php (hunk 1 above)
Comment (radeno) [reply windows-azure-storage.php]: You are right. my mistake, i should use `was_site_meets_php_requirements`

Comment (radeno) [reply windows-azure-storage.php]: Fixed

**Comment by radeno** [comment]
Fixed PR. It was made by a hot needle in Github web app. Sorry, I called incorrect PHP function. It should be `was_site_meets_php_requirements`. Now it is fixed
@ravinderk @faisal-alvi @jeffpaul

**Comment by ravinderk** [comment]
@radeno can you use [wp-compat-validation-tool](https://github.com/10up/wp-compat-validation-tool) library to validate the PHP version?

Sample code: https://github.com/10up/maps-block-apple/pull/174/files

cc: @jeffpaul @faisal-alvi

**Comment by radeno** [comment]
@ravinderk from my point of view this is bigger change. My PR aligned two different PHP versions (7.4 and 8.0) checks currently implemented to just one (8.0). That's all.
Implementing another dependency and removing current logic should not prevent deploying this module as soon as possible.

**Review by ravinderk — APPROVED** [review]

**Pull request merged**